apps/frontend/src/app/2_molecules/SocialLinks/SocialLinks.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import { applyDataAttr, Icon } from '@sovryn/ui';
+
+import { translations } from '../../../locales/i18n';
+import { sovrynLinks } from '../../../utils/constants';
+import styles from './SocialLinks.module.css';
+
+export type SocialLinksProps = {
+  className?: string;
+  innerClassName?: string;
+  dataAttributeId?: string;
+};
+
+const socials = [
+  {
+    id: 'discord',
+    icon: 'discord-logo',
+    url: sovrynLinks.discord,
+  },
+  {
+    id: 'telegram',
+    icon: 'telegram-logo',
+    url: sovrynLinks.telegram,
+  },
+  {
+    id: 'twitter',
+    icon: 'twitter-logo',
+    url: sovrynLinks.twitter,
+  },
+  {
+    id: 'github',
+    icon: 'github-logo',
+    url: sovrynLinks.github,
+  },
+];
+
+export const SocialLinks: FC<SocialLinksProps> = ({
+  className,
+  innerClassName,
+  dataAttributeId,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      className={classNames(styles.socials, className)}
+      {...applyDataAttr(dataAttributeId)}
+    >
+      {socials.map(item => (
+        <a
```

Comment:
I know why you did it but it's a bit weird to use `a` tag if we have `Link` component. I am thinking if we don't need something like `InputBase` that would just provide unstyled link functionality so we could use it with everything and then have `Link` that would just add its styling. Let me know if you think it's an overkill :D

Reply:
yep I was on the fence about this as well, suggestion taken and implemented :)
(it cleans up responsibilities nicely)